react/src/mocks/functions.ts

```diff
@@ -0,0 +1,18 @@
+export function mockMatchMediaQuery(returnBool: Boolean) {
+  // Jest does not implement window.matchMedia() which is used in the chackra ui hook useMediaQuery().
+  // To mock a result of useMediaQuery you have to define this property. The 'matches' boolean is the return value.
+  // https://jestjs.io/docs/26.x/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom
+  Object.defineProperty(window, "matchMedia", {
+    writable: true,
+    value: jest.fn().mockImplementation((query) => ({
+      matches: returnBool,
+      media: query,
+      onchange: null,
+      addListener: jest.fn(),
+      removeListener: jest.fn(),
+      addEventListener: jest.fn(),
+      removeEventListener: jest.fn(),
+      dispatchEvent: jest.fn(),
```

Comment:
Where do you get the list of these function names from? Same below for the Auth0 mock

Reply:
please check the link in the comment: https://jestjs.io/docs/26.x/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom